src/panel/pages/timeline/Timeline.fixture.tsx

```diff
@@ -96,7 +96,7 @@ const DevtoolsContextMock: FC<{ events?: typeof defaultEvents }> = ({
 };
 
 export default {
-  basic: (
+  "basic [no snapshot]": (
```

Comment:
Does this automatically skip snapshots?

Reply:
Yes but I think I've found a better way of doing this now.

In your fixture, set this attribute on the element that should be snapshotted - `data-snapshot="true"`.

This way we also trim down the size of our snapshot images!